Trust and compliance: Add privacy section (#97)
pretix/pretix-docs: docs/trust/privacy/index.md

# Privacy

> On the pages in this section, we inform you on how pretix assists you in staying compliant with privacy regulations.

Reviewer: ```suggestion
This section contains information on how pretix assists you in staying compliant with privacy regulations.
```
